pyproject.toml

```diff
@@ -42,12 +42,12 @@ python = "^3.11"
 # Current incompatibility with jupyter-dash & django-plotly-dash
 # https://github.com/plotly/jupyter-dash/issues/75
 # https://github.com/GibbsConsulting/django-plotly-dash/issues/354
-dash = ">=1.15.0,<2.0.0"
+dash = "*"
```

Comment:
❓ 

As highlighted within [the CircleCI base image](https://github.com/noosenergy/noos-docker-images/pull/117), maybe a more defensive strategy to upgrade first to Python `3.11.*`, then upgrade Dash to `2.*`.